server/routes/main/homepage.js

```diff
@@ -1,38 +1,24 @@
 var querystring = require("querystring");
 var nunjucks = require("nunjucks");
 
-var env = require("../../lib/environment");
-
 function urlLocalizer(locale, remixUrl) {
```

Comment:
the code that used this got removed on L12 so this is never used and jshint dies on it